examples/flutter_gallery/lib/demo/material/date_and_time_picker_demo.dart

```diff
@@ -181,15 +181,17 @@ class _DateAndTimePickerDemoState extends State<DateAndTimePickerDemo> {
                   });
                 },
               ),
+              new Container(height: 8.0),
```

Comment:
It hardly matters but. Typically we use `const SizedBox(height: 8.0)` for a spacer like this